lms/djangoapps/grades/course_grade.py

```diff
@@ -33,6 +42,8 @@ def __init__(self, user, course_data, percent=0.0, letter_grade=None, passed=Fal
         self.letter_grade = letter_grade or None
         self.force_update_subsections = force_update_subsections
 
+        self.last_updated = last_updated
```

Comment:
Post approval comment :-) would it make sense to force the value to GMT here so that you don't need to worry about setting it correctly in other places (there are some adjustments in the related Credentials PR)